execute_pipeline.py

```diff
@@ -37,35 +37,62 @@
 
 import argparse
 import ast
+import base64
 import os
+import tempfile
+import time
+import uuid
 import yaml
 
-from taskflow import engines, task
+from gcloud import storage
+from gcloud import logging
+from taskflow import engines, task, states
 from pipeline.pipelines import pipeline_factory
-from pipeline.utils import job_util
+from pipeline.utils import job_util, pipeline_util
 
 
 def main():
-  pipeline_name, pipeline_kwargs, remote_mode = _parse_args()
-  pipeline = pipeline_factory.make_pipeline(pipeline_name, **pipeline_kwargs)
+  pipeline_name, pipeline_kwargs, env, local_repo = _parse_args()
+
+  if local_repo:
+    pipeline_kwargs = _load_local_repo(local_repo, **pipeline_kwargs)
+
+  if env != 'local':
```

Comment:
It's better to avoid negated conditionals. Here I think it's better to say `if env is 'remote'`.

Also -- these 'local' and 'remote' should be declared as constants rather than immediate values.